bigtable/client/table_admin.h

```diff
@@ -190,6 +190,28 @@ class TableAdmin {
       bigtable::ClusterId const& cluster_id,
       bigtable::SnapshotId const& snapshot_id);
 
+  /**
+   * Generates consistency token for a table.
+   *
+   * @param table_id the id of the table for which we want to generate
+   * consistency token.
+   * @return the consistency token for table.
+   * @throws std::exception if the operation cannot be completed.
+   */
```

Comment:
Can you file a bug to create an example snippet for this operation and to link it from here?